src/GitHub.InlineReviews/Peek/InlineCommentPeekResult.cs

```diff
@@ -22,10 +22,11 @@ public InlineCommentPeekResult(InlineCommentPeekViewModel viewModel)
 
         public Action<IPeekResult, object, object> PostNavigationCallback => null;
 
-        public event EventHandler Disposed;
+        public event EventHandler Disposed = delegate { };
 
         public void Dispose()
         {
+            Disposed?.Invoke(this, EventArgs.Empty);
```

Comment:
I don't think `= delegate { };` is needed as `Disposed` is actually called.

Reply:
All use of `delegate { }` gone now.